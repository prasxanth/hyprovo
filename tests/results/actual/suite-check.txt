2021-11-16 09:27:35.440 - INFO - Pass ✓... (suite-check): add pass
2021-11-16 09:27:35.442 - ERROR - Fail ✗... (suite-check): sub fail 
                                    Expected: 4
                                    Got: -1
2021-11-16 09:27:35.443 - ERROR - Fail ✗... (suite-check): div fail 
                                    Expected: 3
                                    Got: 5.0
2021-11-16 09:27:35.443 - DEBUG - (suite-check): Test Summary → Total: 3    Pass: 1    Fail: 2
