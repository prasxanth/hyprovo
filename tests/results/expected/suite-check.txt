2021-11-14 11:26:49.969 - INFO - Pass ✓... (suite-check): add pass
2021-11-14 11:26:49.972 - ERROR - Fail ✗... (suite-check): sub fail 
                                    Expected: 4
                                    Got: -1
2021-11-14 11:26:49.974 - ERROR - Fail ✗... (suite-check): div fail 
                                    Expected: 3
                                    Got: 5.0
2021-11-14 11:26:49.975 - DEBUG - (suite-check): Test Summary → Total: 3    Pass: 1    Fail: 2
